src/main/java/io/odpf/firehose/sink/bigquery/converter/MessageRecordConverterWrapper.java

```diff
@@ -0,0 +1,8 @@
+package io.odpf.firehose.sink.bigquery.converter;
+
+import lombok.Data;
+
+@Data
+public class MessageRecordConverterWrapper {
```

Comment:
i wonder wether it is necessary to wrap `MessageRecordConverter` on a data class rather than using it directly, what value does it bring ?

Reply:
this is actually a shared object. So one class sets and other uses.

Reply:
this is a shared object that will be set and used by different classes.